src/ui/frmpreferences.ui

```diff
@@ -463,8 +464,8 @@
              <rect>
               <x>0</x>
               <y>0</y>
-              <width>403</width>
-              <height>196</height>
+              <width>395</width>
+              <height>205</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_3">
```

Comment:
Ah! Sorry, I had missed it.

By the way, now that you made me focus on that file I notice that the names of those pages suck a bit. Could you for now change `page_5` to something like `pageShortcuts` so that, even if from the designer it's a blank page, there is some clue about what it does?

Only for `page_5`, so that this pull request stays clean and we can address the others later.